src/ImageSharp.Drawing/Processing/Processors/Drawing/DrawPathProcessor.cs

```diff
@@ -45,7 +45,7 @@ public IImageProcessor<TPixel> CreatePixelSpecificProcessor<TPixel>(Configuratio
             where TPixel : unmanaged, IPixel<TPixel>
         {
             // offset drawlines to align drawing outlines to pixel centers
-            var shape = this.Shape.Translate(0.5f, 0.5f);
+            var shape = this.Shape.Translate(0.5f, 0.5f).Transform(this.Options.ShapeOptions.Transform);
```

Comment:
[Perf] We should avoid doing a full Transform / Copy of the polygon in the default case:

```suggestion
            var shape = this.Shape.Translate(0.5f, 0.5f);
            if (this.Options.ShapeOptions.Transform != Matrix3x2.Identity)
            {
                shape = shape.Transform(this.Options.ShapeOptions.Transform);
            }
```

Reply:
we don't need to do this as the `Transform()` method does this internally.

Reply:
Cool!